doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_features.rst

```diff
@@ -0,0 +1,176 @@
+=========================
+
+Leveraging the latest advancements in semiconductor manufacturing, nRF54H20 is built using a 22 nm process from GlobalFoundries.
+Smaller process nodes typically result in higher-leakage currents.
+However, this is mitigated by leveraging Fully Depleted Silicon-on-Insulator (FD-SOI) process technology.
+FD-SOI reduces the size of the transistor channel compared to bulk CMOS, resulting in lower leakage currents.
+
+Security is key
+===============
+
+The nRF54H20 SoC has been designed with security as a key feature.
+nRF54H20 provides security functionality as services, implemented centrally in the Secure Domain.
+It includes state-of-the-art security features and is designed to achieve PSA Certified Level 3.
+Physically separated centralized security has several advantages.
+It separates the most important security operations from the rest of the system.
+Because of this separation, there is a clear isolation layer where you interact with the secure services via well-defined APIs.
+It also simplifies the inclusion of tamper sensors and side-channel protection without significantly increasing the cost of the device.
+
+Multi-core architecture
+=======================
+
+The nRF54H20 SoC is partitioned into functional blocks, called domains.
+The domains containing the user-programmable main CPUs and associated functions are called cores.
+Most memory and peripherals can be flexibly allocated to cores at compile time.
+To make this possible, the memory and peripherals are located in a separate area called the Global Domain.
+Security functions are centralized into the Secure Domain.
+
+Asymmetric multiprocessing
+==========================
+
+One of the key methods used to optimize the nRF54H20 SoC for very efficient processing is to include several CPUs and optimize them for different workloads.
+In the nRF54H20 SoC, we are using two different types of CPUs: Arm Cortex-M33, and a RISC-V-compatible CPU designed by Nordic Semiconductor called VPR (pronounced “Viper”).
+However, we also apply optimizations to each instance of each processor, so that CPUs are tailored to their range of tasks.
+This way, you can freely select the CPU that is best suited to the various workloads that make up an application.
+Each CPU runs its own instance of the operating system.
+There is no centralized scheduler that assigns threads to various cores.
+This means that you have a full control over which CPU is used for a given task.
+
+System services
+===============
+
+One key feature of the nRF54H20 is that a lot of functionality is provided as services, implemented by Nordic-provided firmware.
+This means that users access functionality through a software API rather than through a set of registers.
+Using a firmware layer means that functionality can be more advanced and handled at a higher level of abstraction, which increases ease-of-use.
+It is also more flexible and updateable than with the purely hardware approach.
```

Comment:
Perhaps we can elaborate on "purely hardware" approach?

Reply:
I'll drop this